### ISSUE TITLE
Cannot call setName; Gas limit must be at least 21000

### ISSUE THREAD

**Issue by AliSawari** [Morteza3123/web3-react#2]
after connecting the web app to metamask, and trying to set a name, MetaMask does not allow the transaction to take place because of the Gas Limit issue, I think if the app switches to the testnest instead of the mainnet, this could be resolved.

any ideas?

**Comment by Morteza3123** [comment]
Yes , my dapp is on truffle testnet and you should deploy it on your testnet and replace contract address and abi . I will add contract code to deploy
